src/nvim/testdir/Makefile

```diff
@@ -230,4 +230,4 @@ newtestssilent: $(NEW_TESTS)
 %.res: %.vim .gdbinit
 	@echo "[OLDTEST] Running" $*
 	@mkdir -p $(TMPDIR)
-	@/bin/sh runnvim.sh $(ROOT) $(NVIM_PRG) $* $(RUN_VIMTEST) -u NONE -S runtest.vim $*.vim
+	@/bin/sh runnvim.sh $(ROOT) $(NVIM_PRG) $* $(RUN_VIMTEST) -S runtest.vim $*.vim
```

Comment:
maybe @ZyX-I meant for it to be capital `-U NONE`, like https://github.com/janlazo/neovim/blob/0dad25f010db326e3689b136e8385fd03d781b0e/src/nvim/testdir/Makefile#L175 ?